### DISCUSSION TITLE
How to remove old version of subfinder

### DISCUSSION THREAD

**Discussion by ThestaRY7** [projectdiscovery/subfinder#507] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/48352738/155576500-81ed1f5a-fb6e-4a6a-a6cb-74a6b0fb73cc.png)

Not installed through brew

How can I uninstall and what methods can I try？

**Comment by ThestaRY7** [comment]
I prefer to use the latest version, anyone ?

**Comment by ehsandeep** [comment, marked as answer]
@ThestaRY7 you can do `type subfinder` to know the current installed location of subfinder and delete the binary, and install the latest version using the command shared in the readme.

**Reply by ThestaRY7** [reply]
wow!!!Thank you.
